resources/js/mollieBlockIndex.js

```diff
@@ -49,21 +49,24 @@ import ApplePayButtonComponent from './blocks/ApplePayButtonComponent'
             gatewayData.forEach(item => {
                 let register = () => registerPaymentMethod(molliePaymentMethod(useEffect, ajaxUrl, filters, gatewayData, availableGateways, item, jQuery, requiredFields, isCompanyFieldVisible, isPhoneFieldVisible));
                 if (item.name === 'mollie_wc_gateway_applepay'  && !isBlockEditor) {
+                     const isExpressEnabled = item.isExpressEnabled;
```

Comment:
If a new variable has the same name as the object property, you can do just that:

`const {isExpressEnabled} = item;`